mappings/1.12.2#17w43a.tinydiff

```diff
@@ -3043,9 +3043,9 @@ c	net/minecraft/unmapped/C_3020744
 		p	4		duration	
 	m	(Lnet/minecraft/unmapped/C_2348249;)V	m_7971793	buttonClicked	
 		p	1		button	
-	m	(II)V	m_0986658	m_0986658	
-		p	1		p_1	
-		p	2		p_2	
+	m	(II)V	m_0986658	setWidthHeight	
```

Comment:
`setDimensions` or `setSize` would be a bit more concise

Reply:
Yeah setSize seems better here